org.hl7.fhir.convertors/pom.xml

```diff
@@ -3,14 +3,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>ca.uhn.hapi.fhir</groupId>
-        <artifactId>org.hl7.fhir.core</artifactId>
+        <groupId>com.accurids.fhir</groupId>
+        <artifactId>core</artifactId>
         <version>5.1.0-patch1</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <artifactId>org.hl7.fhir.convertors</artifactId>
-    <packaging>bundle</packaging>
+    <artifactId>convertors</artifactId>
+    <packaging>jar</packaging>
```

Comment:
It seems we change all the packaging to jar, why? and are there any implications by doing so?